### PR TITLE
[personal-portfolio] Basic website portfolio

### PR DESCRIPTION
**Pull request by dalisc**
This is a basic website portfolio without images (except profile image). What the website currently has are:

- header with profile picture
- navbar with sticky and active properties
- footer with social icons and links
- main content with clickable links and pictures

### CODE REVIEW

File: portfolio/src/main/webapp/images/photography/fileNames.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+1-fog.jpg	    2-forest.jpg       4-blk71.jpg	     5-dexposed10.jpg
+1-halo.jpg	    2-horse.jpg        4-jewel1.jpg	     5-dexposed2.jpg
+1-house.jpg	    2-silhouette2.jpg  4-jewel2.jpg	     5-dexposed3.jpg
+1-houses1.jpg	    2-sky1.jpg	       4-jewel3.jpg	     5-dexposed4.jpg
+1-houses2.jpg	    2-sky2.jpg	       4-jewel4.jpg	     5-dexposed5.jpg
+1-lembangfault.jpg  2-sunrise1.jpg     4-jewel5.jpg	     5-dexposed6.jpg
+1-mountains.jpg     2-sunrise2.jpg     4-lightleak.jpg	     5-dexposed7.jpg
+1-people.jpg	    3-boats.jpg        4-mrt.jpg	     5-dexposed8.jpg
+1-silhouette.jpg    3-cliff.jpg        4-universitytown.jpg  5-dexposed9.jpg
+1-sunrise.jpg	    3-mountain.jpg     4-vivocity1.jpg	     fileNames.txt
```
Comment (123cristen) [inline portfolio/src/main/webapp/images/photography/fileNames.txt]: should fileNames.txt be in here? Also, where is this file being used? If not used, you can remove it :)

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -1,17 +1,263 @@
...
+        <p>
+          This personalized recommendation engine was developed for a virtual
+          hackathon organised by Circles.Life. It uses collaborative filtering
+          to cluster similar users together and content-based filtering to
+          recommend specific items to users in a cluster. We used various APIs
+          to do semantic analysis on event and movie descriptions to see how
+          closely they match users' existing tastes and preferences.
+        </p>
+
+        <a href="" target="_blank"><h3>PokkaPet (Hackomania 2019)</h3></a>
```
Comment (123cristen) [inline portfolio/src/main/webapp/index.html]: should there be a link here?

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -1,17 +1,263 @@
...
+        <h1>My Projects</h1>
+        <p>
+          <span class="">I</span> love getting my hands dirty and making
+          software. I'm always excited to join hackathons or do interesting
+          projects!
+        </p>
+
+        <h2>Tech Hackathons</h2>
+
+        <a href="" target="_blank"><h3>SpotNow (Hack.Asia 2019)</h3></a>
```
Comment (123cristen) [inline portfolio/src/main/webapp/index.html]: should there be a link here?

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -1,17 +1,263 @@
...
+          ><i class="fas fa-envelope-open-text"></i
+        ></a>
+          <span class="tooltiptext">Email me</span>
+        </div>
+
+
+      </div>
+    </footer>
+
+    <script>
```
Comment (123cristen) [inline portfolio/src/main/webapp/index.html]: Consider moving this to a separate javascript file :)

File: portfolio/src/main/webapp/style.css [hunk 5]
```diff
@@ -1,9 +1,184 @@
...
+body {
+  margin: 0;
+  display: grid;
+  grid-template-columns: min-content 1fr;
+  font-family: system-ui, sans-serif;
+  color: #10375c;
+  background-color: #f4f6ff;
+
+  /* this breaks position sticky in Firefox */
+  /* overflow-x: hidden; */
```
Comment (123cristen) [inline portfolio/src/main/webapp/style.css]: should this be removed?

**Review by 123cristen — CHANGES_REQUESTED** [review]
Awesome photos! Also your projects sound really cool :)

File: portfolio/src/main/webapp/images/photography/fileNames.txt (hunk 1 above)
Comment (dalisc) [reply portfolio/src/main/webapp/images/photography/fileNames.txt]: I was just using it to populate an array! I'll be deleting all unused files :)

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (dalisc) [reply portfolio/src/main/webapp/index.html]: Yep, just added!

File: portfolio/src/main/webapp/index.html (hunk 4 above)
Comment (dalisc) [reply portfolio/src/main/webapp/index.html]: I had a problem where when i moved it and imported it the javascript ended up not working :( i'll look into this issue and try again

Comment (dalisc) [reply portfolio/src/main/webapp/index.html]: i have moved it to a new file. turns out i had to wrap in a window.onload function :D

**Review by 123cristen — APPROVED** [review]

**Pull request merged**